inc/compatibility/fixes/class-acceleratedmobilepagesfix.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Accelerated Mobile Pages Fix
+ *
```

Comment:
Nice-to-have: I would like to see a link to the fixes in our docs somewhere in a code comment. It seems to me to make sense that that would live in the fix itself. The reason being, if the docs change, it could help to keep us honest in updating the mu-plugin code accordingly and make it easier to compare the documented solution with the implementation in the mu-plugin. 🔢